docs/sources/clients/promtail/configuration.md

```diff
@@ -183,6 +183,11 @@ Loki:
 # Example: http://example.com:3100/loki/api/v1/push
 url: <string>
 
+custom_headers:
+  # Custom HTTP headers to add to the request.
+  # Example: X-Scope-OrgID: 1
```

Comment:
You give the example of `X-Scope-OrgID` header here. However, this header is added when specifying the `tenant_id`, as documented in line 191-194 in this configuration.md file.
So that may be confusing for users.